### DISCUSSION TITLE
DJL Build Failed：Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

### DISCUSSION THREAD

**Discussion by xuhongxin** [deepjavalibrary/djl#2855] [Development]
i use` ./gradlew build`,but report error

![image](https://github.com/deepjavalibrary/djl/assets/1044109/4e47e6e6-9483-4916-8afb-99c917317da0)

**Comment by frankfliu** [comment]
@xuhongxin 

It's just a warning message, which task it failed?

**Reply by frankfliu** [reply]
Can you check the gradle version with the following command:

```
./gradlew --version
```

**Reply by frankfliu** [reply]
By the way, if you only want to generate .jar files, you can do the following:

```
./gradlew jar
or
./gradlew build -x test
```

**Reply by xuhongxin** [reply]
`./gradlew -verison` result is 
![image](https://github.com/deepjavalibrary/djl/assets/1044109/abf789a1-69ad-4d0c-9266-abe244c74e81)

and the failure task is 
`> Task :jacoco:testCodeCoverageReport FAILED
`

and the most import problem is my intellij idea seems can not really load the project module,because there is no blue icon 
😄
![image](https://github.com/deepjavalibrary/djl/assets/1044109/d8225cad-b692-43ca-81ca-b52b8dd21ec1)

**Comment by frankfliu** [comment]
Can you check your gradle settings?

<img width="1117" alt="image" src="https://github.com/deepjavalibrary/djl/assets/24425551/2066c932-9259-496c-b191-ce3f4b1b8535">

**Reply by xuhongxin** [reply]
![image](https://github.com/deepjavalibrary/djl/assets/1044109/3bb1a882-af77-4df4-bbd2-cdc61f3552ad)

thank you i have success build the project; the core reason is :
1. inteliJ idea jdk is not jdk11;
2. not install gradle plugins;
3. i use network proxy casues gradle jar can not downlode ;

fix all of above and with the help of you,i success build the project; very happy;haha~;
